src/api/goods.ts

```diff
@@ -10,10 +10,12 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => [...goods].sort(
+      (a, b) => a.name.localeCompare(b.name),
+    ).slice(0, 5));
 };
 
 export const getRedGoods = () => {
```

Comment:
Consider using a more descriptive name for the function `getRedGoods`. For example, `getGoodsByColor`. 🤔